pkg/registry/batch/cronjob/strategy.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
```

Comment:
Not sure if you want to pull this out into another PR or not, but would merge faster.

Reply:
eh... if this drags on, I might, but hopefully it won't